best-practices/database/README.md

```diff
@@ -1,4 +1,9 @@
-# DATABASE
+# Databases
+A database is an organized collection of data, generally stored and accessed electronically from a computer system. Where databases are more complex they are often developed using formal design and modeling techniques. 
+
+At Shift3, we generally use T-SQL and NoSQL databases in our projects. More information can be found [here](https://www.altexsoft.com/blog/business/comparing-database-management-systems-mysql-postgresql-mssql-server-mongodb-elasticsearch-and-others/).
+
+Feel free to contact architect@bitwiseindustries.com if you are deciding on which database to use.
```

Comment:
I would word this more strongly. New projects are supposed to go through a pass with an architect.

Reply:
How does:
```suggestion
If you and your team is starting a new project or deciding on a database to use, contact architect@bitwiseindustries.com.
```
look. That way its more explicit as well as covering two common scenarios.

Reply:
Changed via: f277d483354669343f0b1c7c57e69313d9e782d7

Reply:
If you and your team are starting a new project or deciding on a database to use, contact architect@bitwiseindustries.com.

Reply:
Grammar is important. 

Fixed via: a1397fe413e144cbcba9e709633b68536fbeb2e1